source/RevitLookup/Core/ComponentModel/Descriptors/EnumerableDescriptor.cs

```diff
@@ -36,6 +37,7 @@ public EnumerableDescriptor(IEnumerable value)
         IsEmpty = value switch
         {
             string => true,
+            MacroManager enumerable => enumerable.Count == 0,
```

Comment:
Move it to the very bottom. In the top position you should insert frequently used types to increase the efficiency of resolving switching cases